tsdb/engine/tsm1/compact.go

```diff
@@ -77,10 +76,10 @@ type tsmGeneration struct {
 }
 
 // size returns the total size of the generation
-func (t *tsmGeneration) size() int64 {
-	var n int64
+func (t *tsmGeneration) size() uint32 {
```

Comment:
Ok, I actually want to talk through how the different sequences within a generation get created. Realized this morning that I wasn't quite understanding it. I'm not sure how multiple files in the same generation can exist at the same time since only one compaction runs at a time and it always collapses generations.